### PR TITLE
IGNITE-18344 Missing .gitignore items

### PR DESCRIPTION
**Pull request by ibessonov**
https://issues.apache.org/jira/browse/IGNITE-18344

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,9 @@ work
 .DS_Store
 .flattened-pom.xml
 
+**/generated
```
Comment (rpuch) [inline .gitignore:6]: Is this about `build/generated`? If yes, it's already covered by the `build/` being ignored

Comment (ibessonov) [reply .gitignore:6]: No, it's about generated source in module directories. They appear after compilation using Gradle

**Comment by ibessonov** [comment]
Configuring IDEA project with IDEA build process is, apparently, a bad idea. I'm closing the PR

**Comment by ibessonov** [comment]
Re-qualified as a jvm crash dump ignore issue

**Review by sashapolo — APPROVED** [review]

**Review by rpuch — APPROVED** [review]

**Pull request merged**